### DISCUSSION TITLE
How to check the default mappings ?

### DISCUSSION THREAD

**Discussion by teto** [sxyazi/yazi#869] [1. Q&A]
I've been using yazi early on I would say. At the beginning the keymap.toml copied in ~/.config/yazi would override the default one if I understood correctly.
Now they are merged so I am trying to trim down my configuration to only my customizations.
To check the yazi bindings, I have to check https://github.com/sxyazi/yazi/blob/main/yazi-config/preset/keymap.toml

I wish I didn't have to go online to find out the mappings. How to do that ?
Is there a loglevel that could dump the mappings ?
Could the installer dumps the presets ? Right now the nix derivation only writes these files
```
➜ tree /nix/store/sxnlbvxhj1m74rwksxfs1ri1basrknff-yazi-0.2.4/        
/nix/store/sxnlbvxhj1m74rwksxfs1ri1basrknff-yazi-0.2.4/
├── bin
│   └── yazi
└── share -> /nix/store/210174qppd580vl9gw98hipv2l900b27-yazi-0.2.4/share

3 directories, 1 file
```

**Comment by lpnh** [comment]
It seems to me the help menu might have what you're looking for, you can open it by pressing `~`.

**Reply by teto** [reply]
didnt know about that one. Pretty cool ! I would have expected the default mapping to be on `f1` / `?` or `h` though. There is even search available. Ty !
